[new product] rancher (#4073)
captn3m0/endoflife.date: products/rancher.md

---
title: Rancher

> category: service

Reviewer: ```suggestion
category: server-app
```

`service` is reserved for managed service for now (maybe not the best name). We should be using the same tag as https://endoflife.date/argo-cd for now.

> tags: suse rancher kubernetes

Reviewer: ```suggestion
tags: suse
```

We try to only add tags that have chances to be reused, with the exception of company names developing the product. So this only leave the `suse` tag which is the company behind rancher.

iconSlug: rancher
permalink: /rancher
alternate_urls:
-   /suse-rancher

> releasePolicyLink: https://github.com/rancher/rancher/releases

Reviewer: > this should be an official rancher site instead of the github repo, i guess?

Yes. A documentation on GitHub is OK, but it should explain how Rancher is released and supported.

Author: changed the link to the suse rancher lifecycle page

releaseDateColumn: General Availability
activeSupportColumn: End of Maintenance
eolColumn: End of Life
eolWarnThreshold: 121

identifiers:
-   purl: docker:rancher/rancher

> changelogTemplate: "https://github.com/rancher/rancher/releases/tag/v__LATEST__"

Reviewer: ```suggestion
changelogTemplate: https://github.com/rancher/rancher/releases/tag/v__LATEST__
```

auto:
-   git: https://github.com/rancher/rancher.git

# EOL dates can be found on https://www.suse.com/lifecycle#rancher
releases:
-   releaseCycle: "2.7"
    releaseDate: 2022-11-16
    support: 2024-05-15
    eol: 2024-11-18
    latest: "2.7.9"
    latestReleaseDate: 2023-10-26

> -   releaseCycle: "2.6"

Reviewer: ```suggestion

-   releaseCycle: "2.6"
```

    releaseDate: 2021-08-31
    support: 2023-03-01
    eol: 2023-11-30
    latest: "2.6.13"
    latestReleaseDate: 2023-06-01

---

> [Rancher](https://www.rancher.com/) is a complete software stack for teams
adopting containers. It addresses the operational and security challenges of managing multiple
Kubernetes clusters, while providing DevOps teams with integrated tools for running containerized

> workloads.

Reviewer: A description giving an overview of how Rancher is released and supported is needed:

- the versioning policy, if available: looks like it is following semver, but this needs to be confirmed with an official source,
- the release policy, if available: is there something to be said about the release cadence, such as saying that the release cadence is fixed ? 
- the support policy: what does End of Maintenance / End of Life mean ? Is there an official link explaining this (which could be used as the `releasePolicyLink`) ?

Author: couldn't find any official release cadence or policy.
i've added informations about the versioning and definitions of ga, eom and eol.

---
title: Rancher
category: server-app
tags: suse
iconSlug: rancher
permalink: /rancher
alternate_urls:
-   /suse-rancher

> releasePolicyLink: https://www.suse.com/lifecycle#rancher

Reviewer: ```suggestion
releasePolicyLink: https://www.suse.com/support/kb/doc/?id=000020432
```

Maybe we could use this link instead ?

releaseDateColumn: General Availability
activeSupportColumn: End of Maintenance

> eolColumn: End of Life

Reviewer: ```suggestion
activeSupportColumn: Full Support
eolColumn: Limited Support
```

See https://www.suse.com/support/kb/doc/?id=000020435. Description must be updated accordingly.

eolWarnThreshold: 121

identifiers:
-   purl: docker:rancher/rancher

> changelogTemplate: https://github.com/rancher/rancher/releases/tag/v__LATEST__

Reviewer: This line should be moved after `releasePolicyLink` as per [our style guide](https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide).

auto:
-   git: https://github.com/rancher/rancher.git

# EOL dates can be found on https://www.suse.com/lifecycle#rancher
releases:
-   releaseCycle: "2.7"
    releaseDate: 2022-11-16
    support: 2024-05-15
    eol: 2024-11-18
    latest: "2.7.9"
    latestReleaseDate: 2023-10-26

-   releaseCycle: "2.6"
    releaseDate: 2021-08-31
    support: 2023-03-01
    eol: 2023-11-30
    latest: "2.6.13"
    latestReleaseDate: 2023-06-01

# See https://ranchergovernment.com/support-and-maintenance-terms-of-service
# 'Important Definitions' for GA, EOM and EOL definitions

> # 'Understanding Release Versions' for versioning informations

Reviewer: ```suggestion
```

Such comments must be included in the description. Beside, what is ranchergovernment.com ? Between this and [Suse Rancher](https://www.rancher.com/support) and the [open source version of rancher](https://github.com/rancher/rancher) I am a bit lost.

Author: this was the only page i could found containing informations about the definitions. the kb link looks better to me.

---

> [Rancher](https://www.rancher.com/) is a complete software stack for teams
adopting containers. It addresses the operational and security challenges of managing multiple
Kubernetes clusters, while providing DevOps teams with integrated tools for running containerized
workloads.

> SUSE Rancher product releases follows [Semantic Versioning](https://semver.org/).

Reviewer: ```suggestion
SUSE Rancher product releases follows [Semantic Versioning](https://semver.org/).
```

Do you want to document the commercial "SUSE Rancher" product, or the open source version ? If it's the open source one I think we should avoid SUSE Rancher / SUSE Rancher product and just say "Rancher".

Upon the General Availability (GA) date, products are supported and maintained until the End of Maintenance date.
After a product release reaches its End of Maintenance (EOM) date, no further code-level maintenance will be provided, except for critical security- related fixes on a per-request basis.

> Once a product release reaches its End of Life (EOL) date, the customer may continue to use the product within the terms of product licensing agreement.

Reviewer: Again is this the commercial product (then _product licensing agreement_ is OK) or the open source one ?

Author: removed the last line here. the page should fit for the open source rancher.

> Further informations can be found at the official [rancher support matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/).

Reviewer: ```suggestion
Compatibility with various Kubernetes distributions can be found on [the rancher support matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/).
```

Author: i guess we should also change this part to full- and limited support instead of eom/ eol when we change the column names? maybe copied the two descriptions from the kb document.
